sirius.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Alex Damaratski (uixdess)",
+        "oem": "Xiaomi",
+        "device": "Mi 8 SE",
+        "filename": "crDroidAndroid-12.0-20220226-sirius-v8.2.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/sirius/8.x/crDroidAndroid-12.0-20220226-sirius-v8.2.zip/download",
+        "timestamp": 1645897907,
+        "md5": "0cbc2000d598810f88f3bb68412f3093",
+        "sha256": "8bb945fbf8e3497fba28fa45b258a04eaca62a85f9ef9c3375856e7cf712d4b2",
+        "size": 1022239697,
+        "version": "8.2",
+        "buildtype": "Monthly",
+        "forum": "",
```

Comment:
forum?

Reply:
do we have an example for the thread?

Reply:
https://github.com/crdroidandroid/crdroid_features/blob/12.0/ForumPostTemplate